### PR TITLE
Create POM for product selection in Agama

### PR DESCRIPTION
**Pull request by coolgw**
Create POM for product selection in Agama
https://progress.opensuse.org/issues/132884

VR: https://openqa.opensuse.org/tests/3475893#

### CODE REVIEW

File: tests/full-disk-encryption.spec.ts [hunk 1]
```diff
@@ -14,10 +15,12 @@ test.describe('The main page', () => {
 
     test('Full-disk encryption', async ({ page }) => {
         await test.step("Select the product", async () => {
-            await page.getByLabel('SUSE ALP Server').check();
-            await page.getByRole("button", { name: "Select" }).click();
+		const productSelectionPage = new ProductSelectionPage(page);
+		await productSelectionPage.checkSuseAlpServer();
```
Comment (lemon-suse) [inline tests/full-disk-encryption.spec.ts]: We will get the product name from environment variable, refer to ticket https://progress.opensuse.org/issues/133562 and my PR https://github.com/jknphy/e2e-agama-playwright/pull/9

Comment (coolgw) [reply tests/full-disk-encryption.spec.ts]: If outside just call checkSuseAlpServer, the name change will not impact the test logic.

For inside product page, we have to add extra four envs:
process.env.PRODUCTNAME_ALP=
process.env.PRODUCTNAME_MICRO=
process.env.PRODUCTNAME_TW=
process.env.PRODUCTNAME_LEAP=

What's your suggestions?  @lemon-suse @rakoenig

Comment (lemon-suse) [reply tests/full-disk-encryption.spec.ts]: We won't have micro any more, 'SUSE ALP Dolomite' don't have the product selection, only TW and Leap16 has the product selection. Why need so many environment variables? We just need one to set the current product name.

Comment (manfredi) [reply tests/full-disk-encryption.spec.ts]: I am using this:
```
await page.getByLabel(process.env.PRODUCTNAME).check(); 
await page.getByRole("button", { name: "Select" }).click();
```

**Comment by jknphy** [comment]
will be done in #15 thanks!, we based on your work there to continue.

**Pull request closed without merging**